ARDemos/ARDemos/Views/ObjectNode.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+import ARKit
+import SceneKit
+
+class ObjectNode: SCNNode {
+
+    func addObjectNode(nodeName: String, assetPath: String) {
+        guard let node = getNodeFromScene(nodeName: nodeName, assetPath: assetPath) else {
+            return
```

Comment:
don't you want to add preconditionFailure in these returns so we will know if something goes wrong?

Reply:
I mean we could, but one of the issues for this repo (which I will be working on next week since I am on pt) is to add alerts for the users. I really do not think is necessary to crash the app because one node could not be loaded.

Reply:
Good to notify the user but would be nice to think of giving some kind of feedback when something fails on the code execution so devs would be aware of it. 

Just something to keep in mind.

Reply:
I agree, in this case though this is not a code issue. The ARDemo project is supposed to be for designers to just use the app by adding their models. If they have not followed the instructions correctly on how to set the JSON this error of node not found will happen. Which is why I want to create a bunch of error alerts for the user. Because the user is the designer who uploaded the models.